### PR TITLE
sink/codec(cdc): add message size check and support schema default

### PR DESCRIPTION
**Pull request by zhangyangyu**
### What problem does this PR solve?

Issue Number: ref #5338 

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by 3AceShowHand — APPROVED** [review]

**Comment by maxshuang** [comment]
LGTM.
Do you have some e2e tests for the avro default value?

**Review by maxshuang — APPROVED** [review]

File: cdc/sink/codec/avro.go [hunk 1]
```diff
@@ -107,6 +108,20 @@ func (a *AvroEventBatchEncoder) AppendRowChangedEvent(
 		mqMessage.Key = nil
 	}
 	mqMessage.IncRowsCount()
+
+	if mqMessage.Length() > a.maxMessageBytes {
+		log.Error(
+			"Single message too large",
+			zap.Int(
+				"max-message-size",
```
Comment (zhaoxinyu) [inline cdc/sink/codec/avro.go:116]: In cdc, the zap key should be in camel case.

Comment (zhaoxinyu) [inline cdc/sink/codec/avro.go:116]:
```suggestion
				"maxMessageSize",
```

Comment (zhaoxinyu) [inline cdc/sink/codec/avro.go:116]:
```suggestion
				"maxMessageBytes",
```

**Review by zhaoxinyu — APPROVED** [review]

**Pull request merged**